client/app/containers/Blog/index.js

```diff
@@ -55,8 +56,11 @@ export class Blog extends Component {
 
   render() {
     const {
+      currentPage,
       focusedPost,
       loading,
+      maxPages,
+      onGetPosts,
       posts,
       routeParams,
     } = this.props;
```

Comment:
You mean doing something like this instead?

```javascript
render({ currentPage, focusedPost, loading, maxPages, onGetPosts, posts, routeParams }) {
```
Doing so fails and doing {} = this.props is the pattern I've seen; no luck finding examples of the above yet but I'm also not a destructuring expert yet :)